appium-tests/android/android.spec.js

```diff
@@ -1,28 +1,34 @@
-/*jslint node: true, plusplus: true */
+/*jshint node: true */
 /*global beforeEach, afterEach */
```

Comment:
Minor nit: this might be easier to use something like `/*jshint node:true, jasmine: true*/` to avoid declaring all Jasmine globals